ktor-network/jvm/src/io/ktor/network/sockets/JavaSocketAddress.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.network.sockets
+
+import java.lang.reflect.*
+
+public actual sealed class SocketAddress {
+    internal abstract val address: java.net.SocketAddress
```

Comment:
may be it's better to rename it to `jvmAddress` or `jdkAddress`?

Reply:
and also, may be add import aliases to `java.net.*` classes?
(similarly to how kotlinx.datetime do it https://github.com/Kotlin/kotlinx-datetime/blob/master/core/jvm/src/Instant.kt#L22)

Reply:
Thanks for the review. The `address` name was suggested by @e5l.

Reply:
I would rename the file to `SocketAddressJvm` to follow the general pattern and keep `address` as it is

Reply:
@e5l thanks, fixed, also renamed the native file to keep it consistent.